pandas/core/arrays/base.py

```diff
@@ -1443,6 +1443,10 @@ def _reduce(self, name: str, *, skipna: bool = True, **kwargs):
     # Non-Optimized Default Methods; in the case of the private methods here,
     #  these are not guaranteed to be stable across pandas versions.
 
+    def _values_for_json(self) -> np.ndarray:
+        # TODO: document!
```

Comment:
Yeah would be good to have this before merging

Reply:
@WillAyd thoughts on what would be useful here?

Reply:
Guessing this should just be an object dtype array where any value is JSON serializable (ex: string, number, list, dict)

Reply:
the status quo returns non-object ndarrays in many cases

Reply:
Hmm yea but that is also because the JSON serializer has extra logic to handle primitives for those types (eg: int64 backing for datetime). Is the idea here that general EAs need to define this or is this scoped to the internally managed EAs?

Reply:
Gotcha. For that issue @lithomas1 might be interested in implementing the serializer. 

As far as this PR is concerned I am ok with it either way. Generally not sure it should end up in the EA in the long run, but doesn't hurt to move around today either